src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,62 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  value: string,
+  selectValue: string,
+  onChange: (newValue: string) => void,
+  onSelect: (newValue: string) => void
```

Comment:
onChange what?
onSelect what?
Which value?
What is selectValue ?

Use more descriptive names